src/Command/BakeFixtureFactoryCommand.php

```diff
@@ -110,10 +110,13 @@ public function setTable(string $tableName, ConsoleIo $io)
      */
     public function getPath(Arguments $args): string
```

Comment:
I don't understand what the purpose of the `$args` parameter? It does not appear to be referenced inside the function. Can it be removed or does it have a purpose I'm not aware of?

Reply:
See the signature of `parent::getPath()`